src/components/badge/badge.test.js

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import Badge from "./badge.component";
+import Button from "../button/button.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import {
+  badge,
+  badgeCounter,
+  badgeCrossIcon,
+} from "../../../cypress/locators/badge";
+
+const BadgeComponent = ({ ...props }) => {
+  return (
+    <Badge {...props}>
+      <Button
+        style={{
+          marginRight: 0,
+        }}
+        buttonType="tertiary"
+      >
+        Filter
+      </Button>
+    </Badge>
+  );
+};
+
+context("Testing Badge component", () => {
+  describe("Should render Badge component", () => {
```

Comment:
describe("should render Badge component", () => {

Reply:
Changed 'Should' to 'should'